test/converters_and_test_searchspaces/test_sample_configuration_spaces.py

```diff
@@ -57,9 +57,17 @@ def run_test(self):
                 c.is_valid_configuration()
                 neighborhood = ConfigSpace.util.get_one_exchange_neighbourhood(
                     c, seed=i)
-                np.random.shuffle(neighborhood)
-                for n in neighborhood[:5]:
+                # np.random.shuffle(neighborhood)
```

Comment:
Please remove the comments and use the enumerate function to make this code shorter/easier.